boards/beagle/beagleconnect_freedom/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 
 zephyr_library()
 zephyr_library_sources(
-    board_antenna.c
+    ../../ti/cc1352p1_launchxl/board_antenna.c
```

Comment:
would prefer if the file was copied over, having this is just asking for it to silently break if the vendor or board are renamed or if the board is removed

Reply:
Maybe it would be better to extract sky13317 to `drivers/misc`? This is used in cc1352p1_launchxl, beagleconnect_freedom and beagleplay_cc1352.

Reply:
Closing this now since sky13317 is now a separate driver.